src/transformState.js

```diff
@@ -5,7 +5,25 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const obj of actions) {
+    switch (obj.type) {
+      case 'addProperties':
+        Object.assign(state, obj.extraData);
+        break;
+
+      case 'removeProperties':
+        for (const key of obj.keysToRemove) {
+          if (state[key]) {
+            delete state[key];
+          }
+        }
```

Comment:
You used the forEach construction below, then why not to use it here?) It's more clear and readable, and moreover, it will make the code look more flat, and not like this (with ступенями от лестници)

Reply:
That's not that scary to use just common loops instead of array methods))